Sources/SwiftGenKit/Utils/Metadata.swift

```diff
@@ -1,9 +1,7 @@
 //
-//  Metadata.swift
-//  SwiftGenKit
-//
-//  Created by David Jennes on 05/05/2018.
-//  Copyright © 2018 AliSoftware. All rights reserved.
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
```

Comment:
This file could profit from a doc comment (on the `enum Metadata`, or at least on the `static func generate(for data: Any)` fonction) to describe the purpose of it. I have to admit I wondered what it was used for when I reviewed this (I think it's for JSON/YAMLPlist parsers when we describe the structure of a file's content, but given this `Metadata.swift` file is in `Utils/` nothing makes it related to them and it's not clear what this is for at first sight)